Initial layout and headers

neynah opened the pull request:
Will fix ampersand and add other things in the future.

kentonv commented on about.html:
```diff
@@ -0,0 +1,53 @@
+---
+layout: page
+---
+<div id="about">
```
Instead of wrapping `<header>` in `<div>`, assign an ID or class name to `<header>` itself.

kentonv commented on style.scss:
```diff
@@ -222,19 +210,286 @@ body>footer {
+// =======================================================================================
+// how-it-works.html
+
+body>#how-it-works {
```
It looks like there is a lot of similarity between the header styles for each page. We should try to have the common features defined once so that it's easier to make changes.

kentonv commented on style.scss:
```diff
@@ -222,19 +210,286 @@ body>footer {
+    background-color: #1b4460;
+  }
+
+    >h1 {
```
This is nitpicky and not super-important, but you have some inconsistent indentation, and this can make it hard to read the code. Indentation is a lot more visible than the actual `{}` braces, so over time you'll learn to rely on it when jumping around quickly, but if it's inconsistent then you won't be able to rely on it.

kentonv commented on style.scss:
```diff
@@ -76,8 +84,18 @@ body {
 }
 
-h1, h2, h3, h4, h5, h6 {
+h1 {
```
By separating h1 out here, now h1 is bold, because headings are bold by default. But I thought only the word "Sandstorm" in the front-page title was supposed to be bold?

neynah commented:
Thanks for the feedback! I:
- added more content
- updated links
- cleaned up most of the indenting
- consolidated some of the header styling that was repeated across multiple page headers into one place

I'll keep working on this, let me know if you have more suggestions/feedback.

kentonv commented:
howitworks-left.svg lists Mesos twice. :)

On that image, I think I'd also like to make the dots a bit smaller and more clustered far to the left, to make it clearer that we're illustrating the huge gap between them and Sandstorm.

Let's arrange the dots vaguely like this:

Vaguely upper-left: Plain Linux, VMs (non-containers)
Vaguely lower-left: OpenStack, AWS (non-container orchestration)
Vaguely upper-right: Docker, rkt (containers)
Vaguely lower-right: Mesos, Kubernetes (container orchestration)

By "vaguely" I mean there shouldn't be any clear visual separation between these quadrants, since we're not actually measuring them on any particular scale. Just nice to have similar products "sort of together".

I'm going to merge this pull request as-is, but let's update the image in a follow-on PR.
